src/bccontext_fabric_io.rs

```diff
@@ -0,0 +1,89 @@
+extern crate thiserror;
+extern crate wapc_guest as guest;
+
+use crate::BitcodeContext;
+
+use std::io::{ErrorKind, Read, SeekFrom};
+
+pub struct FabricSteamReader<'a> {
```

Comment:
I hope this is just a typo...
```suggestion
pub struct FabricStreamReader<'a> {
```